browser_extension/js/main.js

```diff
@@ -283,13 +362,27 @@ $(function () {
     updateTyping();
   });
 
+
+
   // Click events
 
   // Launches thethe Google login flow
   $googleSignInButton.click(function () {
     loginGoogleUser();
   });
 
+  $logout.click(function () {
```

Comment:
Let's hold off on the logout functionality for now. At least until we get a solid login process. It will make it easier to debug